src/topp/CometAdapter.cpp

```diff
@@ -283,8 +317,10 @@ class TOPPCometAdapter :
     isotope_error["0/1"] = 1;
     isotope_error["0/1/2"] = 2;
     isotope_error["0/1/2/3"] = 3;
-    isotope_error["-8/-4/0/4/8"] = 4;
-    isotope_error["-1/0/1/2/3"] = 5;
+    isotope_error["-1/0/1/2/3"] = 4;
+    isotope_error["-1/0/1"] = 5;
+    isotope_error["-3/-2/-1/0/1/2/3"] = 6;
+    isotope_error["-8/-4/0/4/8"] = 7;
```

Comment:
these strings are also used in the registerOptionsAndFlags_ for the isotope_error parameter.
Can we define them once and use there and here?

Reply:
e.g., one could define a map from index to string and use that in both places?
I did not think it through but could work